src/utils/system.cpp

```diff
@@ -205,7 +205,7 @@ System::System(int argc, char* argv[])
         else        verbose_embrakes = true;
         break;
       case 'C':
-        strncpy(config_file, optarg, 250);
+        strncpy(config_file, optarg, 250-1);
```

Comment:
What's the reason for subtracting by one here?

Reply:
This is to account for `\0` termination character in strings. Otherwise, a warning about string size is produced.